clock.rb

```diff
@@ -0,0 +1,13 @@
+require 'clockwork'
+require './config/boot'
+require './config/environment'
+
+include Clockwork
```

Comment:
please ensure we're logging to a place that gets rotated (such as log/production.log)